lib/config-initializer.js

```diff
@@ -26,6 +26,22 @@ function writeFile(config, isJson, callback) {
         callback(e);
         return;
     }
+
+    // install any external configs as well as any included plugins
+    if (config.extends && config.extends.indexOf("eslint") === -1) {
```

Comment:
Within our current code, we only ever set this to `"eslint:recommended"` (or `"airbnb"`, etc) which I think makes this safe for now.

If we do want to support the array syntax do you have a specific suggestion for how to manage doing multiple `exec` calls? As of now we're not using `async`, but I'd probably either want to use that or something like https://www.npmjs.com/package/exec-sync.